views/proposals.tmpl

```diff
@@ -133,29 +144,46 @@
                                         {{if eq $vr.Option.OptionID "yes"}}
                                             {{$votesPercent :=  percentage $vr.VotesReceived $v.TotalVotes}}
                                             {{if lt $.Tip.Height (toInt $v.Endheight)}}
-                                                <span class="text-progress position-relative" data-tooltip="Vote Status (approval %)">
+                                                <span class="d-none d-sm-inline text-progress position-relative" data-tooltip="Voting in progress">
```

Comment:
This is actually the most important column, IMO, so I'd like to leave this with full text if possible.

Reply:
this is done